README.md

```diff
@@ -29,3 +29,31 @@ The app is built with Ruby on Rails 7. It uses `importmap-rails`, `turbo-rails`,
 
 ## Setup dev environment
 See [`.env.sample`](.env.sample) for required/optional environment variables.
+
+## Setup dev environment with docker
+```bash
+$ cp .env.sample .env
+```
+
+Generate required environment variables
+```bash
+$ docker exec -it conference-app rails c
```

Comment:
`docker compose` looks more correct. What do you think?

```suggestion
$ docker compose exec -it conference-app rails c
```

Reply:
@unasuke 
I see. Thanks for the feedback!